stacks/devel.json

```diff
@@ -127,7 +127,9 @@
       },
       "FROM": "rocker/geospatial:devel",
       "ENV": {
-        "NB_USER": "rstudio"
+        "NB_USER": "rstudio",
+        "VIRTUAL_ENV": "/opt/venv",
+        "PATH": "${PATH}:${VIRTUAL_ENV}/bin"
       },
```

Comment:
This would only sets these new envs on the devel image (i.e. image based on R-devel) of geospatial.  I think we really want to set these in any of the images that call `install_python.sh` (or `install_jupyter.sh`) in their RUN block, on both devel and 4.3.2 (aka latest) stack files, right? i.e. on https://github.com/rocker-org/rocker-versioned2/blob/master/stacks/4.3.2.json, for every stack entry that includes a call to `install_python.sh`, right?  (apologies @eitsupi if I've miss-remembered where these definitions get set in the current build process!)

Reply:
I agree I'd like it to be on all the jsons :) I had initially thought that it would need to be here and the other JSON stuff is autogenerated off this by make (hence me opening #720), but I am probably wrong? :D Should I just manually add it to all the json files?

Reply:
When we run this script, the two latest stack files will be updated. For older versions, please update them manually.

https://github.com/rocker-org/rocker-versioned2/blob/56ce3b840cf7f79a809676d09372a71519f21c47/build/make-stacks.R#L503